build.gradle

```diff
@@ -2,6 +2,15 @@ import org.gradle.internal.os.OperatingSystem
 
 // to update the gradle wrapper, execute ./gradlew wrapper --gradle-version 4.0
 
+buildscript {
+    repositories {
+        mavenCentral()
```

Comment:
I am wondering: Isn't jcenter (the site we have been using so far) a superset of maven central? And we remember that we used maven central before and then replaced it with jcenter, because of the availability of some library.

Why do you add maven central here?

Reply:
Yes, agree, jcenter is superior. However, we could add it as second repo, in case some libs are not found on jcenter

Reply:
This whole buildscript tag is just a workaround since as of now junit 5 does not support the id-syntax, ie something like
̀ ```
plugins {
id 'junit5' version '5'
}
````